src/plugins/intel_gpu/src/plugin/sync_infer_request.cpp

```diff
@@ -734,16 +740,15 @@ std::vector<cldnn::event::ptr> SyncInferRequest::prepare_input(const std::string
     }
 
     cldnn::event::ptr ret_event = nullptr;
-    if (!is_remote) {
-        if (convert_needed) {
-            convert_and_copy(user_tensor.get(), device_tensor.get(), stream);
-        } else {
-            auto src_ptr = static_cast<uint8_t*>(user_tensor->data());
-            if (!same_host_mem(memory, src_ptr)) {
-                ret_event = memory->copy_from(stream, src_ptr, false);
-            }
+    if (!is_remote && !convert_needed) {
+        auto src_ptr = static_cast<uint8_t*>(user_tensor->data());
+        if (!same_host_mem(memory, src_ptr)) {
+            ret_event = memory->copy_from(stream, src_ptr, false);
         }
     }
+    if (convert_needed) {
+        convert_and_copy(user_tensor.get(), device_tensor.get(), stream);
```

Comment:
You need to use `convert_and_copy(const ov::ITensor* src, cldnn::memory::ptr dst, cldnn::stream& stream)`, otherwise it may try to lock memory of usm_device buffer for writing (we can't access usm_device from the host side)